src/state/atoms/localSearchAtom.ts

```diff
@@ -1,6 +1,5 @@
 import { atom } from 'jotai';
 import { Orama, create, insert } from '@orama/orama';
-
```

Comment:
Just a small nit and not blocking, but I'd avoid making changes like this. It creates extra noise and the review "looks bigger" that it really is.